test/server/admin/clusters_handler_test.cc

```diff
@@ -107,6 +108,7 @@ TEST_P(AdminInstanceTest, ClustersJson) {
   {
    "name": "fake_cluster",
    "observability_name": "observability_name",
+   "eds_service_name": "potato_launcher",
```

Comment:
Why is only the JSON path validated and not the text one?

Reply:
It also checks txt format. [link](https://github.com/envoyproxy/envoy/pull/22902/files/8fbf0445fec2e1ec0dc549194cc85c62c048d824#diff-7fc8d177a372e19dab401678b8126faea43dff984156aad5e39ca74a795f9bebR223) The test name is not very correct.